pa/private.go

```
@@ -0,0 +1,410 @@
+					},
+				})
+
+				return true
+			})
+		}
+		if !found {
+			continue
+		}
+
+		debugf("rewriting %s", fileRelPath)
+		if err := os.WriteFile(fileRelPath, []byte(sprintCode(root)), 0o644); err != nil {
```

golangci-lint

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)